src/data-workspace/title-bar.js

```diff
@@ -0,0 +1,25 @@
+import i18n from '@dhis2/d2-i18n'
+import { IconDimensionDataSet16 } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { StatusTag } from '../shared/status-tag/index.js'
+import styles from './title-bar.module.css'
+
+const TitleBar = ({ workflow }) => (
```

Comment:
Might make more sense to have `TitleBar` take props `workflowName`, `workflowDataSetsCount` and `workflowApprovalState` instead :thinking:

Reply:
Yeah, I'd probably go with either one of these two options:
A. Don't pass any props to titlebar and call `useWorkflowContext()` in the component.
B. Pass three individual props. If going this route, I think the names you suggested are good, but we could omit the `workflow` prefix.